### ISSUE TITLE
Navigate to start/end of a tracklet

### ISSUE THREAD

**Issue by xulman** [mastodon-sc/mastodon#264]
Given the current Menu:
![Screenshot_20231020_150039](https://github.com/mastodon-sc/mastodon/assets/10509335/889e81f0-c122-47a6-a764-14d347e3a2b4)

I propose to add two more "hot spots" on a track:
- Select Tracklet Downward
- Select Tracklet Upward
- Navigate to Tracklet start
- Navigate to Tracklet end

(the latter two could also be: Tracklet mother and Tracklet daughter... not perhaps super clear on the first look, but I believe user will find the difference)

They complement "in-between" the current special targets:
- the intermediate previous/consecutive spot
  - to where one can select or navigate
  - which jumps maximally over one division (to the very first daughter spot)
- *here the new thing*
  - can select and navigate
  - isn't jumping over division points
- and the very beginning(root) / end(all leafs)
  - to where one can only select
  - which jumps over divisions

**Comment by xulman** [comment]
> - Navigate to Tracklet start
> - Navigate to Tracklet end

This functionality is actually already there, try ctrl+alt+up/down... but not visible from the menu, one has to "notice it" in the Keymap settings (filter for *navigate*)



> - Select Tracklet Downward
> - Select Tracklet Upward

Is also there, try ctrl+shift+up/down.. .but ain't working (at least) on WinX (fresh stuff as of 13th March 2024)

-> Update:
Maybe add these to the menu, definitively fix the selection thing (and avoid using Meta keys as there's no key for it on Win platforms)

**Comment by tinevez** [comment]
Yes there are some weirdness with these shortcuts I am not sure I understand. They are not the same in BDV (no need for alt) and in TrackScheme, despite being listed the same in the keymap window. I don't get it.
